source/Willow/ComponentClassificationCommandBuilder.class.st

```diff
@@ -5,50 +5,53 @@ Class {
 	#name : #ComponentClassificationCommandBuilder,
 	#superclass : #GRObject,
 	#instVars : [
-		'styleProviderBinding'
+		'classesProvider'
+	],
+	#pools : [
+		'Willow'
 	],
 	#category : #'Willow-Frontend'
 }
 
 { #category : #private }
-ComponentClassificationCommandBuilder >> commandStyledWith: aCssStyle [
+ComponentClassificationCommandBuilder >> commandStyledWith: aSymbol [
 
-	^ ComponentClassificationCommand toStyleWith: aCssStyle
+	^ ComponentClassificationCommand toStyleWith: classesProvider >> aSymbol
 ]
 
 { #category : #Building }
 ComponentClassificationCommandBuilder >> dialogContainer [
 
-	^ self commandStyledWith: styleProviderBinding content dialogContainer
+	^ self commandStyledWith: #dialogContainer
 ]
 
 { #category : #Building }
 ComponentClassificationCommandBuilder >> disabledComponent [
 
-	^ self commandStyledWith: styleProviderBinding content disabledComponent
+	^ self commandStyledWith: #disabledComponent
 ]
 
 { #category : #Building }
 ComponentClassificationCommandBuilder >> fadingNotification [
 
-	^ self commandStyledWith: styleProviderBinding content fadingNotification
+	^ self commandStyledWith: #fadingNotification
 ]
 
 { #category : #initialization }
 ComponentClassificationCommandBuilder >> initialize [
 
 	super initialize.
-	styleProviderBinding := Binding to: WillowCssStyles
+	classesProvider := Classification
```

Comment:
This should be renamed to classificationProvider or maybe even classificationNamespace if we expect it to be one.